devpmtTestData/deviceReporter/review3/review3.md

```diff
@@ -0,0 +1,6 @@
+== Device reporter files used for 3. review
```

Comment:
It looks like this entire directory is a straight copy of %universal/testData. I'd recommend either i) simply referencing these files from your dependency of universal, or ii) copying just the subset which are in active use in this project

Reply:
I had originally duplicated this because work and metadata that Javi was working on was in a long running branch, which was needed for this.  I've swapped back to reading from universal now.